# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 30

# Position Controllers ---------------------------------------
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: left_joint
  pid: {p: 50, i: 100, d: 1.0}
joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: right_joint
  pid: {p: 50, i: 100, d: 1.0}
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: endeffector
  pid: {p: 50, i: 100, d: 1.0}
joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: armToEnd
  pid: {p: 50, i: 100, d: 1.0}
joint5_position_controller:
  type: effort_controllers/JointPositionController
  joint: topToBot
  pid: {p: 50, i: 100, d: 1.0}
joint6_position_controller:
  type: effort_controllers/JointPositionController
  joint: shoulderTolittle
  pid: {p: 50, i: 100, d: 1.0}
joint7_position_controller:
  type: effort_controllers/JointPositionController
  joint: botToBase
  pid: {p: 50, i: 100, d: 1.0}

